lib/v2/gamegene/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'gamegene.cn/': {
```

Comment:
```suggestion
    'gamegene.cn': {
```
Domain name should not have trailing slash.

docs: https://docs.rsshub.app/joinus/new-radar